tests/test_service/test_adapter.py

```diff
@@ -552,3 +554,27 @@ def move_tif(*args, **kwargs):
 
         # Ensure container clean-up was requested:
         mock_rmtree.assert_called_once_with(self.temp_dir)
+
+    @patch('harmony_service.adapter.download')
+    def test_forbidden_download(self, mock_download):
+        mock_download.side_effect = ForbiddenException('You are forbidden to download.')
+        message = Message(
+            {
+                'accessToken': self.access_token,
+                'callback': 'https://example.com/',
+                'sources': [{'collection': 'C1234-EEDTEST', 'shortName': 'test'}],
+                'stagingLocation': self.staging_location,
+                'user': self.user,
+                'format': {'mime': 'image/png'},
+            }
+        )
+
+        hybig = BrowseImageGeneratorAdapter(
+            message, config=self.config, catalog=self.input_stac
+        )
+
+        with self.assertRaisesRegex(
+            HyBIGServiceError,
+            ('You are forbidden to download.'),
+        ):
+            hybig.invoke()
```

Comment:
This tests that an error raised in the harmony-service-lib is raised as a HyBIGServiceError and has the correct information attached. This is displayed in the workflow-ui